devito/ir/iet/analysis.py

```diff
@@ -145,41 +144,54 @@ def mark_vectorizable(analysis):
 def mark_wrappable(analysis):
```

Comment:
note for reviewers: this is the function that determines whether TAO is applicable or not